wordpress-mobile/WordPress-iOS
# Leverage PostAutoUploadInteractor support for offline pages.

This PR leverages the code in `PostAutoUploadInteractor` to also support pages.  This isn't wired though, since the interactor is used by the post coordinator, which doesn't support pages yet.

Additionally: this PR implements several unit tests for page messaging.  These tests required the interactor to work with pages, so they sort of validate the changes mentioned above.

### To test:

Run the unit tests and make sure that they succeed. Double check their definition.

### PR submission checklist:
